src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -2246,6 +2276,15 @@ function processInputData<T extends number | string>(
             seriesGradientColorscale?.length > 0
               ? markerSymbolGradient
               : markerSymbol(index),
+          // need to set marker.line for a transparent case (opacity != 1)
```

Comment:
The `opacity: 0.7` a few lines above here is being applied to the final plot, so when the user chooses 1 on the slider, they actually only get 0.7.

The solution may just to remove this line.

Reply:
@bobular Thank you for your suggestion. I will remove it! 👍